templates/includes/install-problems.hbs

```diff
@@ -2,28 +2,42 @@
+      <li><h4>Repeat the flash process.</h4>Up to five tries have been reported to possibly give a successful result in the end.</li>
+    </ul>
+    <h3>Boot gets stuck even after repeated reflash</h3>
+    Congratulations, you might have found a new issue for us to solve. This should not happen when using stable builds. The nightlies are however advancing quickly and it is not always possible for us to test implications on all watches accordingly. Since the watches install pages link to the nightlies currently, you likely used those. You can try again and <a href="https://release.asteroidos.org/">install a stable build when available</a>.<br>To help us solve the problem, you could try to collect logs. It is possible that the watch does not show a UI but is already accessible using <code>ssh ceres@192.168.2.15</code> or <code>adb shell</code> in the state the boot process stops. In case ssh or ADB connections work, try the following commands to write logs to local .txt files.
+    <ul>
+      <li><code>ssh root@192.168.2.15 "dmesg" > dmesg-watch.txt</code></li>
+      <li><code>ssh root@192.168.2.15 "journalctl" > journal-watch.txt</code></li>
+      <li><code>adb shell logcat > logcat-watch.txt</code></li>
+    </ul>
+    In case no connection is available, it is still interesting to see if the watch leaves any traces in your local dmesg. Try to spot USB related messages in <code>sudo dmesg -w</code> while the watch is connected and booting. Record the log using the <code>sudo dmesg > dmesg-local.txt</code> command.<br>
+    You can share those logs in a new <a href="https://github.com/AsteroidOS/meta-smartwatch/issues">Github issue</a> tagged with the watch codename. For support chat, you are invited to <a href="https://matrix.to/#/#Asteroid:matrix.org">join our Matrix channel</a>.
+    <h3>Watch only boots when powered via USB, but shuts off when booted while worn</h3>
+    This is a clear sign of a worn battery that will soon fail. Maybe you should replace it. The boot process of AsteroidOS uses all cores of a watch to ensure a fast startup. This leads to power spikes during the boot process, which can completely shut down watches with worn out batteries. It has been observed that WearOS can still boot on watches with worn out batteries, but AsteroidOS cannot, depending on the progress of battery wear. However, AsteroidOS can usually still provide a useful 10-20 hours of uptime when the watch is worn once booted while powered up. Watch batteries are relatively easy to replace on watches with screwed down backs. Glued back plates usually require a more or less unruly process of heating and prying.
```

Comment:
`more or less unruly` is strange. I'm not entirely sure what should go here. maybe words like `frustrating process`, `compromised water resistance` and `creative solutions involving hair driers` should be involved, but I'm not sure how much info should go here